README.md

```diff
@@ -38,5 +40,7 @@ with [Meteor](https://www.meteor.com).
 [docker_image]: https://hub.docker.com/r/mquandalle/wekan/
 [heroku_button]: https://www.herokucdn.com/deploy/button.png
 [heroku_deploy]: https://heroku.com/deploy?template=https://github.com/wekan/wekan/tree/master
+[indiehosters_button]: https://indie.host/signup.png
```

Comment:
Arf, sorry you were right... the dns is still propagating... I'll try again tomorrow, and ping you